packages/gatsby/cache-dir/loader.js

```diff
@@ -315,6 +315,16 @@ export class BaseLoader {
     return page && page.notFound === true
   }
 
+  isPageExist(rawPath) {
+    const pagePath = findPath(rawPath)
+    const page = this.pageDb.get(pagePath)
+    return (
+      page &&
+      stripSurroundingSlashes(page.payload.page.path) ===
+        stripSurroundingSlashes(rawPath)
```

Comment:
also question about usage of those `stripSurroundingSlashes` - this shouldn't be needed I think? We don't really use that in actual navigation code elsewhere (from my check) so it's bit of red flag that this is here

Reply:
handled